sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/AbstractSpannerFn.java

```diff
@@ -22,12 +22,15 @@
 import com.google.cloud.spanner.Spanner;
 import com.google.cloud.spanner.SpannerOptions;
 import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.util.ReleaseInfo;
 
 /**
  * Abstract {@link DoFn} that manages {@link Spanner} lifecycle. Use {@link
  * AbstractSpannerFn#databaseClient} to access the Cloud Spanner database client.
  */
 abstract class AbstractSpannerFn<InputT, OutputT> extends DoFn<InputT, OutputT> {
+  private static final String USER_AGENT_PREFIX = "apache.beam.java";
```

Comment:
According to https://tools.ietf.org/html/rfc7231#page-46
format should be like `User-Agent: CERN-LineMode/2.15 libwww/2.17b3`

We just want to make sure to track the usage of the library.